modules/plugins/diagnostics/nvim-lint/config.nix

```
@@ -14,6 +14,18 @@ in {
       startPlugins = ["nvim-lint"];
       pluginRC.nvim-lint = entryAnywhere ''
         require("lint").linters_by_ft = ${toLuaObject cfg.linters_by_ft}
+        local linters = require("lint").linters
+        local custom_linters = ${toLuaObject cfg.custom_linters}
+        for linter, config in pairs(custom_linters) do
+          linters[linter] = config
+        end
+
+        local builtin_linters = ${toLuaObject cfg.builtin_linters}
+        for linter, config in pairs(builtin_linters) do
+          for key, val in pairs(config) do
+            linters[linter][key] = val
+          end
+        end
```

```suggestion
        local linters = require("lint").linters
        local nvf_custom_linters = ${toLuaObject cfg.custom_linters}
        for linter, config in pairs(nvf_custom_linters) do
          linters[linter] = config
        end

        local nvf_builtin_linters = ${toLuaObject cfg.builtin_linters}
        for linter, config in pairs(nvf_builtin_linters) do
          for key, val in pairs(config) do
            linters[linter][key] = val
          end
        end
```

Less likely to conflict with user specified Lua, which might choose to use "custom_linters." Also distinguishes clearly where the linters and formatters come from.